src/tests/gobprepare/cloner/test_oracle_to_postgres.py

```diff
@@ -193,47 +193,126 @@ def test_copy_data(self):
             call("tabledef_c"),
         ])
 
+    def test_list_to_chunks(self):
+        testcases = [
+            (([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 3), [{"min": None, "max": 4}, {"min": 4, "max": 7}, {"min": 7, "max": 10}, {"min": 10, "max": None}]),
+            (([1, 2, 3], 1), [{"min": None, "max": 2}, {"min": 2, "max": 3}, {"min": 3, "max": None}]),
+            (([], 4), []),
+            (([1, 3], 2), [{"min": None, "max": None}]),
+            (([1, 3], 5), [{"min": None, "max": None}]),
+        ]
```

Comment:
probably hypothetical but what if the chunk_size = 0?

Reply:
Raises an error now :)